src/main/java/edu/iastate/ece/sd/sdmay2126/GUIForm.java

```diff
@@ -91,6 +95,56 @@ public void run() {
                                         Thread.sleep(15000);
 
                                         // TODO: Interact with the narrative (fill form, click buttons, read output)
+                                        //Identify the third textbox and clicks it note if the textboex are rearanged on kbase this will break
+                                       WebElement codeBox =  new WebDriverWait(driver, 10).until(d -> d.findElements(By.cssSelector("div[class^='cell code_cell']"))).get(2);
```

Comment:
Did you consider using a CSS class selector?

`By.cssSelector(".cell.code_cell")`